tests/test_model.py

```
@@ -1261,3 +1258,274 @@ def test_can_init_bayesian_neural_network_mo_cc(
+# CategoricalFeatureConfig
+# ---------------------------------------------------------------------------
+
+
+def test_categorical_feature_config_basic():
+    cfg = CategoricalFeatureConfig(vocab_size=3)
+    assert cfg.vocab_size == 3
+    assert cfg.embedding_dim == 8  # default
+
+
+def test_categorical_feature_config_invalid_vocab_size():
+    with pytest.raises(Exception):
```

Ruff (0.15.4)

[warning] Do not assert blind exception: `Exception`

(B017)